.github/workflows/deploy-to-s3.yml

```
@@ -0,0 +1,102 @@
+name: Deploy to S3
+
+on:
+  push:
+    branches:
+      - dev
+    tags:        
+      - '*' 
+
+jobs:
+  install_and_build:
+    runs-on: ubuntu-latest
+
+    env:
+      CI: false
+
+    environment: ${{ (github.ref == 'refs/heads/dev' && 'dev') || (startsWith(github.ref, 'refs/tags/v') && 'UN') }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Cache dependencies
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.pnpm-store
+          key: ${{ runner.os }}-pnpm-${{ hashFiles('**/pnpm-lock.yaml') }}
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Install pnpm
+        run: npm install -g pnpm@8.6.0
+
+      - name: pnpm version
+        run: pnpm --version
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: 'Create env file'
+        run: |
+          touch .env
+          echo REACT_APP_ENCRYPTED_ENDPOINT=${{ vars.REACT_APP_ENCRYPTED_ENDPOINT}} >> .env
+          echo REACT_APP_QRCODE_VERIFY_ENDPOINT=${{ vars.REACT_APP_QRCODE_VERIFY_ENDPOINT }} >> .env
+          echo REACT_APP_SCHEMA_URL=${{ vars.REACT_APP_SCHEMA_URL }} >> .env
+          echo REACT_APP_REMOTE_AGENT_API_KEY=${{ vars.REACT_APP_REMOTE_AGENT_API_KEY }} >> .env
+          echo REACT_APP_DEFAULT_AGENT_ID=${{ vars.REACT_APP_DEFAULT_AGENT_ID }} >> .env
+          cat .env
+
+      - name: Build explorer
+        run: pnpm build:js
+
+      - name: Archive artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: explorer
+          path: packages/demo-explorer/build/explorer
+
+  deploy_to_s3:
+    needs: install_and_build
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+
+    environment: ${{ (github.ref == 'refs/heads/dev' && 'dev') || (startsWith(github.ref, 'refs/tags/v') && 'UN') }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: explorer
+          path: packages/demo-explorer/build/explorer
+
+      - name: Install AWS CLI
+        run: |
+          sudo apt-get -q install -y python3-pip
+          pip3 install awscli --upgrade --user
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v3
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_GITHUB_ACTION_ARN }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Deploy to S3
+        run: aws s3 sync packages/demo-explorer/build/explorer/ s3://${{ vars.AWS_S3_BUCKET_NAME }} --delete
```

we need to make sure that the FE works properly when deployed under the`/vckit/explorer/` path
so we need to put files under `/vckit/explorer/` in s3 bucket and generate static files to take into account that path
```suggestion
        run: aws s3 sync packages/demo-explorer/build/explorer/ s3://${{ vars.AWS_S3_BUCKET_NAME }}/vckit/explorer/ --delete
```